docs/virtual_machines/disks_and_volumes.md

```diff
@@ -1101,67 +1101,120 @@ spec:
-[vhostmd](https://github.com/vhostmd/vhostmd).
+The `downwardMetrics` exposes a limited set of VM and host metrics to the
+guest. The format is compatible with [vhostmd](https://github.com/vhostmd/vhostmd).
 
 Getting a limited set of host and VM metrics is in some cases required to allow
 third-parties diagnosing performance issues on their appliances. One prominent
 example is SAP HANA.
 
-Example:
-
-    apiVersion: kubevirt.io/v1
-    kind: VirtualMachineInstance
-    metadata:
-      labels:
-        special: vmi-fedora
-      name: vmi-fedora
-    spec:
-      domain:
-        devices:
-          disks:
-          - disk:
-              bus: virtio
-            name: containerdisk
-          - disk:
-              bus: virtio
-            name: metrics
-        machine:
-          type: ""
-        resources:
-          requests:
-            memory: 1024M
-      terminationGracePeriodSeconds: 0
-      volumes:
-      - name: containerdisk
-        containerDisk:
-          image: kubevirt/fedora-cloud-container-disk-demo:latest
-      - name: metrics
-        downwardMetrics: {}
-
-The `vm-dump-metrics` tool can be used to read the metrics:
-
-    $ dnf install -y vm-dump-metrics
-    $ vm-dump-metrics
-    <metrics>
-      <metric type="string" context="host">
-        <name>HostName</name>
-        <value>node01</value>
-    [...]
-      <metric type="int64" context="host" unit="s">
-        <name>Time</name>
-        <value>1619008605</value>
-      </metric>
-      <metric type="string" context="host">
-        <name>VirtualizationVendor</name>
-        <value>kubevirt.io</value>
-      </metric>
-    </metrics>
+In order to expose the `downwardMetrics` to the VMs, there are two methods supported: `disk` and `virtio-serial port`.
 
 > **Note:** The **DownwardMetrics** feature gate
 > [must be enabled](../operations/activating_feature_gates.md#how-to-activate-a-feature-gate)
-> to use this volume. Available starting with KubeVirt v0.42.0.
+> to use the metrics. Available starting with KubeVirt v0.42.0.
+ 
+#### Disk
+
+A volume is created, and it is exposed to the guest as a raw block volume.
+
+Example:
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-fedora
+  name: vmi-fedora
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: metrics
+    machine:
+      type: ""
+    resources:
+      requests:
+        memory: 1024M
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - name: containerdisk
+    containerDisk:
+      image: quay.io/containerdisks/fedora:latest
+  - name: metrics
+    downwardMetrics: {}
+```
+
+#### Virtio-serial port
+
+This method uses a virtio-serial port to expose the metrics data to the VM. 
+It creates a port named `/dev/virtio-ports/org.github.vhostmd.1` inside the VM, from where the metrics can be retrieved.
```

Comment:
Make it more clear that KubeVirt implements/supports the Virtio Transport protocol of vhostmd over this port?

Reply:
Done!